build/templates/olm-artifacts-template.yaml.tmpl

```diff
@@ -65,4 +65,25 @@ objects:
         name: dedicated-admin-operator
         source: dedicated-admin-operator-registry
         sourceNamespace: openshift-operator-lifecycle-manager
-  
\ No newline at end of file
+    - apiVersion: monitoring.coreos.com/v1
+      kind: PrometheusRule
+      metadata:
+        labels:
+          prometheus: sre-dedicated-admin-alerts
+        name: sre-dedicated-admin-alerts
+        namespace: openshift-monitoring
+      spec:
+        groups:
+        - name: sre-dedicated-admin-alerts
+          rules:
+          - record: DedicatedAdminDesiredBlackList:count
+            expr: |
+              count(kube_namespace_created{namespace =~ "^(kube|openshift)-.*"}) + 
```

Comment:
How do we ensure this stays in sync with the values in the codebase? Is there another way we can achieve this, in code, so that we don't have to duplicate the default regexes here?

Reply:
I had a talk with @jewzaam  on how to sync that, since it's somewhat static data we would filter it by PR review initially.

I created an Issue so we don't lose track of that: https://github.com/openshift/dedicated-admin-operator/issues/50

Also added a comment to the section where we maintain the regexp in the code, indicating we must reflect any change in the `PrometheusRule`

Reply:
What if there was an exposed metric that correlated to what we want here? That is, we care about when there's a +1 to the metric in the blocked namespace metric. The proposed metric would move the concern to into the operator which so it's the only place the namespace exists.

Reply:
How we manage the regex for this will change very soon so we can support a more generic setup.  There are 6 total Groups we need to support in the near future instead of the one we support now and they're going to be for various use cases, some overlapping.  I'm OK with having this hard coded for now and deal with it in the future.

Reply:
@jewzaam I left a note about this in the Google doc...